### PR TITLE
Add the specification committee operations doc

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

File: content/commitees/specification/operations.md [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+TCK challenge process, and the compatibility request process. Creating
+an issue template is described in this GitHub doc:
+<https://help.github.com/en/articles/creating-issue-templates-for-your-repository>.
+Creating labels is described in this GitHub doc:
+<https://help.github.com/en/articles/creating-a-label>.
+
+The current TCK challenge and compatibility request process requires the
+following issue labels, which should be defined in the issue tracker:
+
++-----------------------------------+-----------------------------------+
```
Comment (autumnfound) [inline content/commitees/specification/operations.md]: This table doesn't render correctly on the current version of Hugo (https://deploy-preview-590--jakartaee.netlify.com/commitees/specification/operations/). We've done an upgrade that has slightly shifted how Markdown is processed to be more in line with how [GitHub processes markdown](https://github.github.com/gfm/#tables-extension-), so you'll need to make a couple of tweaks to the table text to get it to render properly. If you like, I can swing by and help you make the changes you need!

Comment (ivargrimstad) [reply content/commitees/specification/operations.md]: Thanks! I saw that and fixing it now...

**Comment by ivargrimstad** [comment]
@autumnfound Thanks! Who can merge this PR to get it published?

**Comment by autumnfound** [comment]
That'll be @chrisgiundon! He should sweep through in the next day or so

**Pull request merged**